Lambda-School-Labs/mydish-fe
# Feature/unit text input

This PR changes the `Unit` input to be a `TextInput` element.

**To test:** 
1. Create a new recipe.
2. Try entering whatever the heck you want as a unit (with a max length of 11*).
3. Try navigating through the inputs using the `Return` key, which should move your cursor to the next field, though not to the next _line_ of fields.

This PR also adjusts some styling in `CreateRecipeForm`. Nothing major.

* I came up with 11 because that's the length of "tablespoons," which is probably the longest unit anyone would ever enter.
